staging/src/k8s.io/component-base/version/verflag/verflag.go

```diff
@@ -28,25 +28,25 @@ import (
 	"k8s.io/component-base/version"
 )
 
-type versionValue int
+type VersionValue int
 
 const (
-	VersionFalse versionValue = 0
-	VersionTrue  versionValue = 1
-	VersionRaw   versionValue = 2
+	VersionFalse VersionValue = iota
```

Comment:
How about doing away with the iota altogether? Let's have these constants be strings.
All we do is translate these between their string representations and then compare them. Not worth all the translation code in my opinion.

Reply:
K, changed `VersionValue` to `string` in the new version of the patch I just pushed